R/release.r

```diff
@@ -243,13 +245,21 @@ cran_submission_url <- "http://xmpalantir.wu.ac.at/cransubmit/index2.php"
 #' @export
 #' @keywords internal
 submit_cran <- function(pkg = ".") {
-  pkg <- as.package(pkg)
-  maint <- maintainer(pkg)
-  comments <- cran_comments(pkg)
+  built_path <- build_cran(pkg)
+  upload_cran(pkg, built_path)
+}
 
+build_cran <- function(pkg) {
   message("Building")
   built_path <- build(pkg, tempdir(), manual = TRUE)
   message("File size: ", file.info(built_path)$size, " bytes")
```

Comment:
Or make it always pretty: https://github.com/gaborcsardi/prettyunits (sorry, again, self-promotion). If you don't want another dependency, the function is very small, so you can just take it: https://github.com/gaborcsardi/prettyunits/blob/master/R/sizes.r